WordPress/src/main/java/org/wordpress/android/viewmodel/activitylog/ActivityLogViewModel.kt

```diff
@@ -511,6 +535,21 @@ class ActivityLogViewModel @Inject constructor(
         return true
     }
 
+    private fun onDownloadBackupFileClicked(url: String) {
+        activityLogTracker.trackDownloadBackupDownloadButtonClicked(rewindableOnly)
+        _navigationEvents.value = Event(ActivityLogNavigationEvents.DownloadBackupFile(url))
+    }
+
+    private fun dismissNoticeClicked(downloadId: Long) {
+        activityLogTracker.trackDownloadBackupDismissButtonClicked(rewindableOnly)
+        viewModelScope.launch {
+            // reload events first to remove item - convey progress
+            reloadEvents(backupDownloadEvent = BackupDownloadEvent(displayNotice = false, displayProgress = false))
```

Comment:
@ParaskP7 - Please note that I moved the reloadEvents call before the dismiss call because I didn't want to leave room for the button to be tapped more than once. It appears that all UI refreshes call reloadEvents and disabling the button would to have called the method anyway. Plus I tested with a disable and it made the view jumpy because the transition happens quickly.

Reply:
👋 @zwarm !

Thanks for guiding me through your thought process. 🙏